packages/runtime/test/lib/testUtils.ts

```diff
@@ -705,7 +707,7 @@ export async function waitForRecipientToReceiveNotification(
  *
  * Returns the sender's own shared ThirdPartyRelationshipAttribute.
  */
-export async function executeFullRequestAndShareThirdPartyRelationshipAttributeFlow(
+export async function executeFullRequestThirdPartyRelationshipAttributeQueryFlow(
```

Comment:
why have you renamed this testUtil? I don't see a change in the method.

Reply:
Because this function was only used with ReadAttributeRequestItem and ThirdPartyRelationshipAttributeQuery.

And I create a method that used the `ShareAttribureRequestItem` to transfer a ThirdPartyAttribute.

Reply:
IMO it's not important on how a function is used for naming, instead it should describe what it can be used for...